bundles/org.openhab.binding.pihole/src/main/java/org/openhab/binding/pihole/internal/PiHoleHandler.java

```diff
@@ -218,6 +193,19 @@ private void refresh() {
         if (localDnsStatistics.enabled()) {
             updateState(DISABLE_ENABLE_CHANNEL, new StringType(ENABLE.toString()));
         }
+        var gravityLastUpdated = localDnsStatistics.gravityLastUpdated();
+        if (gravityLastUpdated != null) {
+            var absolute = gravityLastUpdated.absolute();
+            if (absolute != null) {
+                var instant = Instant.ofEpochSecond(absolute);
+                var zonedDateTime = instant.atZone(ZoneId.of("UTC"));
+                updateState(GRAVITY_LAST_UPDATE, new DateTimeType(zonedDateTime));
```

Comment:
It would be nice if the set time zone matches the configured localisation settings in openHAB.
You can use the `TimeZoneProvider`

Reply:
I thought it's done automagically by OH

nevertheless - done